### PR TITLE
[Twitter Adapter] Add unit tests for all the Webhooks/Models/Twitter files

### PR DESCRIPTION
**Pull request by sw-joelmut**
## Description
This PR adds unit tests for all the [Webhooks/Models/Twitter](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet/tree/develop/libraries/Bot.Builder.Community.Adapters.Twitter/Webhooks/Models/Twitter) files increasing the code coverage from **8%** to **100%**.

### Specific Changes
- Added **14** files to include **26** new unit tests.

## Testing
The following images show the new tests passing and the resulting code coverage.
image
image

### CODE REVIEW

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/NewDirectMessageModelsTests.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+namespace Bot.Builder.Community.Adapters.Twitter.Tests.Webhooks.Models.Twitter
+{
+    [TestClass]
+    [TestCategory("Twitter")]
+    public class NewDirectMessageModelsTests
+    {
+        [TestMethod]
+        public void EventPropertiesShouldBeSetSuccessfully()
+        {
+            var ev = new Event()
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/NewDirectMessageModelsTests.cs]:
```suggestion
            var event = new Event()
```

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            };
+
+            var jsonResult = JsonConvert.DeserializeObject<WebhookDMResult>(JsonConvert.SerializeObject(webhookDmResult));
+
+            Assert.AreEqual(webhookDmResult.Users.GetType(), jsonResult.Users.GetType());
+            Assert.AreEqual(webhookDmResult.Events.GetType(), jsonResult.Events.GetType());
+        }
+
+        [TestMethod]
+        public void WebhookDMResultShouldConvertoToDirectMessageEventSuccessfully()
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs]:
```suggestion
        public void WebhookDMResultShouldConvertToDirectMessageEventSuccessfully()
```

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        {
+            DirectMessageEvent dmEvent = new WebhookDMResult()
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs]: These are redundant. Please remove them all.
```suggestion
            DirectMessageEvent dmEvent = new WebhookDMResult
```

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                attachment = new Attachment()
+            };
+
+            Assert.AreEqual("test-text", messageData.text);
+            Assert.AreEqual(typeof(TwitterEntities), messageData.entities.GetType());
+            Assert.AreEqual(typeof(Attachment), messageData.attachment.GetType());
+        }
+
+        [TestMethod]
+        public void AtatchmentPropertiesShouldBeSetSuccessfully()
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs]:
```suggestion
        public void AttachmentPropertiesShouldBeSetSuccessfully()
```

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+            };
+
+            Assert.AreEqual("event-id", dmEvent.id);
+            Assert.AreEqual("event-type", dmEvent.type);
+            Assert.AreEqual(3000, dmEvent.created_timestamp);
+            Assert.AreEqual(typeof(Message), dmEvent.message_create.GetType());
+        }
+
+        [TestMethod]
+        public void DmEventInstanceShouldConvertoToDirectMessageResultSuccessfully()
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs]:
```suggestion
        public void DmEventInstanceShouldConvertToDirectMessageResultSuccessfully()
```

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookRegistrationTests.cs [hunk 6]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookRegistrationTests.cs]: Please remove unnecessary using statements.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]
Some details and also, please add the copyright header to all the new files.

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/NewDirectMessageModelsTests.cs (hunk 1 above)
Comment (sw-joelmut) [reply tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/NewDirectMessageModelsTests.cs]: It doesn't let me use the 'event' name, it is a reserved keyword, any other name in mind?

Comment (ceciliaavila) [reply tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/NewDirectMessageModelsTests.cs]: Maybe `twitterEvent`?

**Review by ceciliaavila — APPROVED** [review]

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/MediaEntityTests.cs [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+            Assert.Equal(1, mediaEntity.id);
+            Assert.Equal("id-1", mediaEntity.id_str);
+            Assert.Equal("https://display-url", mediaEntity.display_url);
+            Assert.Equal("https://expanded-url", mediaEntity.expanded_url);
+            Assert.Equal("http://media-url", mediaEntity.media_url);
+            Assert.Equal("https://media-url", mediaEntity.media_url_https);
+            Assert.Equal("https://url", mediaEntity.url);
+            Assert.Equal("media-type", mediaEntity.type);
+            Assert.Equal(3, mediaEntity.indices.Length);
+            // TwitterSizeThumb
```
Comment (santgr11) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/MediaEntityTests.cs:45]: There should be a space before each comment

**Review by ian-whiteout — APPROVED** [review]

**Pull request closed without merging**